### DISCUSSION TITLE
Is sdkman-cli-native ready to be used?

### DISCUSSION THREAD

**Discussion by ssh352** [sdkman/sdkman-cli-native#248] [Q&A] [answered]
I am new to sdkman. Do you recommend using sdkman-cli-native (the Rust version)? Thanks!

**Comment by marc0der** [comment, marked as answer]
Hi @ssh352,

If you are installing SDKMAN via the standard channel from `https://get.sdkman.io`, you are already using it. You can confirm this by looking under `$HOME/.sdkman/libexec`—we [overlay the Rust binaries](https://github.com/sdkman/sdkman-cli-native/discussions/18) on top of the legacy bash commands. We are gradually rolling this out, so they will completely eclipse the bash commands over time.
